harpy/_validations.py

```
@@ -252,6 +252,26 @@ def vcf_samplematch(vcf, bamlist, vcf_samples):
         sys.exit(1)
     return(query)
 
+def vcf_contig_match(contigs, vcf):
+    vcf_contigs = []
+    vcf_header = subprocess.run(["bcftools", "head", vcf], capture_output = True, text = True)
+    for i in vcf_header.stdout.split("\n"):
+        if i.startswith("##ALT="):
+            break
+        if i.startswith("##contig="):
+            unformatted_contig = i.split(",")[0]
+            vcf_contigs.append(unformatted_contig.lstrip("##contig=<ID="))
```

Ruff

Using `.strip()` with multi-character strings is misleading

(B005)